sktime/forecasting/statsforecast.py

```diff
@@ -552,3 +555,255 @@ def get_test_params(cls, parameter_set="default"):
+
+    Parameters
+    ----------
+    season_length : Union[int, List[int]]
+        Number of observations per unit of time. For multiple seasonalities use a list.
+    trend_forecaster : estimator, optional, default=AutoETS()
+
+
+    References
+    ----------
+    .. [1]
+        https://nixtla.github.io/statsforecast/src/core/models.html#multiple-seasonalities
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.statsforecast import StatsForecastMSTL
+
+    >>> y = load_airline()
+    >>> model = StatsForecastMSTL(season_length=[3,12])
+    >>> fitted_model = model.fit(y=y)
+    >>> y_pred = fitted_model.predict(fh=[1,2,3])
+    >>> y_pred
+    1961-01    454.507079
+    1961-02    419.341442
+    1961-03    477.373918
+    Freq: M, Name: Number of airline passengers, dtype: float64
+    """
+
+    _tags = {
+        "ignores-exogeneous-X": True,
+        "capability:pred_int": True,
+        "capability:pred_int:insample": True,
+    }
+
+    from sktime.forecasting.ets import AutoETS
+
+    def __init__(
+        self,
+        season_length: Union[int, List[int]],
+        trend_forecaster=AutoETS(),
+    ):
+        self.season_length = season_length
+        self.trend_forecaster = trend_forecaster
+
+        super().__init__()
+
+    def _instantiate_model(self):
+        """Create underlying forecaster instance."""
+        from statsforecast.models import MSTL
+
+        from sktime.forecasting.base._base import BaseForecaster
+
+        if issubclass(type(self.trend_forecaster), BaseForecaster):
+            self.trend_forecaster = StatsForecastBackAdapterMSTL(self.trend_forecaster)
+
+        return MSTL(
+            season_length=self.season_length,
+            trend_forecaster=self.trend_forecaster,
+        )
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        from sktime.forecasting.theta import ThetaForecaster
+
+        del parameter_set  # to avoid being detected as unused by ``vulture`` etc.
+
+        params = {
+            "season_length": [3, 12],
+            "trend_forecaster": ThetaForecaster(),
+        }
```

Comment:
Can you please add few more test configurations?

1. corresponding to `season_length` being `int` instead of `typing.List[int]`
2. corresponding to the case where forecaster is multivariate in nature, e.g. `VAR`